crmsh/bootstrap.py

```diff
@@ -180,12 +182,54 @@ def _validate_nodes_option(self):
         """
         Validate -N/--nodes option
         """
-        if not self.node_list:
+        if self.type == "join":
+            self.current_user = userdir.getuser()
+            if self.cluster_node.find('@') != -1:
+                user, node = self.cluster_node.split('@')
+                self.user_list = [user]
+                self.cluster_node = node
+            else:
+                self.user_list = [userdir.getuser()]
```

Comment:
I think `userdir.getuser()` no need to be called twice here, how about
```
self.user_list = [self.current_user]
```

Reply:
Sure)